orm/types/ormerrors/errors.go

```diff
@@ -13,8 +13,8 @@ func IsNotFound(err error) bool {
 }
 
 var (
-	InvalidTableId                = errors.New(codespace, 1, "invalid or missing table or single id, need a non-zero value")
-	MissingPrimaryKey             = errors.New(codespace, 2, "table is missing primary key")
+	ErrInvalidTableID             = errors.New(codespace, 1, "invalid or missing table or single id, need a non-zero value")
```

Comment:
This is API breaking, i don't think we should rename them.

Reply:
reverted